src/openzaak/components/catalogi/api/filters.py

```diff
@@ -86,6 +86,11 @@ class RolTypeFilter(FilterSet):
         field_name="zaaktype__identificatie",
         help_text=get_help_text("catalogi.ZaakType", "identificatie"),
     )
+    omschrijving = filters.CharFilter(
+        field_name="omschrijving",
+        lookup_expr="icontains",
+        help_text=_("Filter op omschrijving (case-insensitive)."),
```

Comment:
Let's make sure this is marked experimental in the OAS
```suggestion
        help_text=mark_experimental("Filter op omschrijving (hoofdletterongevoelig)."),
```